src/components/posts/GenderGraphContainer.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { LineChart, Line, XAxis, YAxis, CartesianGrid, Tooltip, ResponsiveContainer } from 'recharts';
+
+export default function GenderGraphContainer() {
+
+    const data = [
```

Comment:
Instead of putting the hardcoded graph data inside of the component itself, it might clean up our codebase better if it's separated into a constant .js file that we can import in instead, similar to how the `initialAds.js` file is right now.